core/pkg/opencost/allocation.go

```diff
@@ -942,6 +946,21 @@ func (a *Allocation) RAMEfficiency() float64 {
 	return 1.0
 }
 
+// GPUEfficiency is the ratio of usage to request. Note that, without the NVIDIA
+// DCGM exporter providing Prometheus with usage metrics, this will always be
+// zero, as GPUUsageAverage will be zero (the default value).
+func (a *Allocation) GPUEfficiency() float64 {
+	if a.GPURequestAverage > 0 && a.GPUUsageAverage > 0 {
```

Comment:
Unlikely to matter, but the other efficiency funcs have a nil check on `a`. Might be good to add that.

Reply:
Good point, I'll add that in now.

Reply:
Fixed!